protobuf/daemon/login.proto

```diff
@@ -4,6 +4,21 @@ package pb;
 
 option go_package = "github.com/NordSecurity/nordvpn-linux/daemon/pb";
 
+enum LoginType {
+  LoginType_UNKNOWN = 0;
```

Comment:
[small] can it ever be `UNKNOWN`?

Reply:
it's grpc common/recommended practice to have `None` value e.g. when creating variable of that type to not get assigned some valid value which is not intended.